client/src/components/existing-request/search-pending.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <MainContainer id="search-pending-container">
+    <template v-slot:container-header>
+      <v-col cols="auto h4">
+        You are searching for an existing Name Request
+      </v-col>
+    </template>
+
+    <template v-slot:content>
+      <v-row justify="center">
+        <v-col cols="auto">
+          <b>Please wait a moment</b>
+        </v-col>
+      </v-row>
+      <v-row justify="center">
+        <v-col cols="auto">
+          <v-progress-circular color="orange"
+                               id="analyze-pending-spinner"
+                               size="50"
+                               indeterminate />
```

Comment:
Spacing seems a bit off here.

Reply:
This code indentation isn't my favourite style but I kept it consistent with the file I copied this from (analyze-pending.vue).
Is this what you are referring to, and do you think I should change it? I don't mind.

Reply:
oh yea, i'm totally fine if thats uniform in this app :)